New docs structure (#56)
opengisch/QField-docs: en/prepare/print-to-pdf.rst

Print to PDF
==================================

It's possible to export the maps and other items to a PDF document.

> If the project has at least one print layout defined, they are available over the menu.

Reviewer: ```suggestion
If the project has at least one print layout defined, printing is available over the menu.
```
